test/validate_test.js

```diff
@@ -445,5 +446,35 @@ describe("Validation", () => {
         });
       });
     });
+
+    describe("custom ErrorList", () => {
+      const schema = {
+        type: "string",
+        required: true,
+        minLength: 1
+      };
+
+      const formData = 0;
+
+      const CustomErrorList = ({ errors }) => (
+        <div className="CustomErrorList">{errors.length} custom</div>
+      );
+
+      let compInfo = createFormComponent({
+        schema,
+        liveValidate: true,
+        formData,
+        ErrorList: CustomErrorList
+      });
+      let comp = compInfo.comp;
+      let node = compInfo.node;
```

Comment:
Because code within describe blocks are immediately executed, when running suites, please move the compInfo initialization code in the `it` statement, and just use `const {node}`, eg

```js
it("should use CustomErrorList", () => {
  cont {node} = createFormComponent({
    schema,
    liveValidate: true,
    formData,
    ErrorList: CustomErrorList
  });

  expect(node.querySelectorAll(".CustomErrorList")).to.have.length.of(1);
  expect(node.querySelector(".CustomErrorList").textContent).eql(
    "1 custom"
  );
});
```